common/constant/key.go

```diff
@@ -19,6 +19,8 @@ package constant
 
 type DubboCtxKey string
 
+type TripleCtxKey string
```

Comment:
I suggest that moving this type to protocol/triple/triple_protocol/handler_compat.go since Triple protocol should try not to rely on dubbo-go as much as possible.